### PR TITLE
Fix Quoting of arguments from Jetty `start.jar --dry-run`

### PR DESCRIPTION
**Pull request by joakime**
This addresses the concerns brought up by Issue #10055 with a minimal fix.

### CODE REVIEW

**Comment by joakime** [comment]
This is DRAFT, as more work on `jetty.sh` is being done to avoid re-evaluating the `RUN_ARGS` once they are set, to avoid double-expansion of arguments by the shell.

File: jetty-home/src/main/resources/bin/jetty.sh [hunk 1]
```diff
@@ -499,7 +499,8 @@ case "$ACTION" in
           disown \$!
           echo \$! > \"$JETTY_PID\""
       else
-        echo ${RUN_ARGS[*]} | xargs ${JAVA} > /dev/null &
+        # echo ${RUN_ARGS[*]} | xargs ${JAVA} > /dev/null &
+        echo ${RUN_ARGS[*]} | xargs ${JAVA} & # TODO: don't check in this version only used for cli debugging
```
Comment (joakime) [inline jetty-home/src/main/resources/bin/jetty.sh:503]: This is temporary (just avoiding the redirection to `/dev/null` during testing)

**Comment by joakime** [comment]
After a conversation with @sbordet and @lorban to show them what we are dealing with, it might be possible to write unit tests that actually execute `jetty.sh` in a safe way.

Stay tuned, going to write up a few examples Thursday.

**Review by gregw — CHANGES_REQUESTED** [review]
Why has this forked my branch and lost recent work? There is no need to do things like lookup tables, as this does not need to be super fast, plus it is only used at startup, so likely more time spent building lookup table than using it.

If there is an issue with jetty.sh, then I would prefer that work was done in #10099 and it is fixed, rather than redoing the work on quoting.

**Comment by joakime** [comment]
Closing in favor of #10099

**Pull request closed without merging**